src/components/date/__internal__/date-picker/date-picker.spec.tsx

```diff
@@ -93,7 +127,7 @@ describe("DatePicker", () => {
 
     it(`should pass to the "DayPicker" component the "disabledDays"
         prop containing a null value`, () => {
```

Comment:
question: does this description need updating as you've changed `null` to `undefined` in the `.toEqual()` assertion?